sys/arm64/arm64/locore.S

```diff
@@ -226,6 +237,17 @@ ENTRY(mpentry)
 	bic	x2, x2, CPACR_CEN_MASK
 	orr	x2, x2, CPACR_CEN_TRAP_NONE
 	msr	cpacr_el1, x2
+
```

Comment:
We need an instruction barrier here (`isb`). Without it the `msr` may not have completed before the `mrs` below leading to a capability exception.

Reply:
Do you want to add those as a followup commit in dev after I merge this given you found the issue?  The barrier isn't present yet in morello-dev either, so adding it to dev as a direct commit might make the most sense.

Reply:
Though I guess I can add a new commit in this PR as well.

Reply:
I already prepared the patch (with commit message) yesterday for morello-dev and was just waiting for Andy to test it. Though I think there is merit in having the barrier always be present in dev (there's a later isb that means things currently work in dev, it only becomes a problem when touching cctlr_el0 here because it's done so soon before the next isb) and just fixing up this commit to include it.

Reply:
True, I can just do it as a fixup.